lib/Pikirasa/Exception.php

```diff
@@ -1,4 +1,20 @@
 <?php
 namespace Pikirasa;
 
-class Exception extends \Exception { }
+class Exception extends \Exception
+{
+    public function __construct($message, $code = 0, \Exception $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+
+        if ($openSSlErrorMessage = openssl_error_string()) {
```

Comment:
This `$openSSlErrorMessage` variable is unused below. Either drop it, or use it below.